recipe/meta.yaml

```diff
@@ -10,18 +10,22 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - fix-vasp-py3.patch
 
 build:
-  number: 2
+  number: 0
   script: python setup.py install
```

Comment:
Since `setuptools` was added, this should have been changed to

```
python setup.py install --single-version-externally-managed --record record.txt
```

Reply:
xref: https://github.com/conda-forge/ase-feedstock/issues/16